powerquery-docs/connectors/automy-data-analytics.md

```diff
@@ -1,22 +1,22 @@
 ---
 title: Power Query Automy Data Analytics Connector
 description: Provides basic information and prerequisites for the Automy Data Analytics connector
-author: DougKlopfenstein
+author: Carlos Quintero
 ms.topic: conceptual
-ms.date: 1/8/2024
-ms.author: dougklo
+ms.date: 11/25/2024
+ms.author: cquintero89
```

Comment:
The ms.author name needs to be a Microsoft employee. Revert this change back to "dougklo" (my Microsoft alias).